spyder/widgets/fileswitcher.py

```diff
@@ -658,8 +660,20 @@ def setup_file_list(self, filter_text, current_path):
         self.fix_size(paths)
 
         # Build the text that will appear on the list widget
-        path_text_font_size = CONF.get('appearance', 'rich_font/size', 11)
-        filename_text_font_size = path_text_font_size + 2
+        rich_font = CONF.get('appearance', 'rich_font/size', 11)
+        if sys.platform == 'darwin':
+            path_text_font_size = rich_font
+            filename_text_font_size = path_text_font_size + 2
+        elif os.name == 'nt':
+            path_text_font_size = rich_font - 1
+            filename_text_font_size = path_text_font_size + 1
```

Comment:
Judging from the screenshots, Windows looks a point size or two smaller than the rest. Can you bump it up (by one?) to ensure it matches better?

Reply:
@ccordoba12 what do you think about this?

Reply:
I'm thinking to reduce Linux rich font size by one point, so I think Windows font size is fine.

Reply:
@CAM-Gerlach, you can adjust it to your own prefs, this is just the default.

Reply:
@ccordoba12 Yes, but its controlled by the global rich text size for *everything* in the UI, so it needs to be reasonably sized *relative* to the rest of the UI text. On Windows, it is, so if we bring Linux and macOS down one point as you state, then everything should be fine.

Reply:
I'm ok in doing that for Linux but I don't know about macOS.

@juanis2112, how things look for you in mac?

Reply:
@juanis2112, to change the size of the rich font on Linux, please go to this line

https://github.com/spyder-ide/spyder/blob/master/spyder/config/fonts.py#L44

and change that value to `10`.

Then go to this line

https://github.com/spyder-ide/spyder/blob/master/spyder/config/main.py#L765

and change that value to `47.6.0`

Reply:
I think we could reduce size in macOS 1 point too.

Reply:
@juanis2112, you forgot to bump CONF_VERSION, as I described it here:

> Then go to this line
>
> https://github.com/spyder-ide/spyder/blob/master/spyder/config/main.py#L765
>
> and change that value to `47.6.0`

Reply:
@ccordoba12 You are right! Sorry. I just did a new PR with that change.

Reply:
You are right! Sorry

Sure, no problem :+1:

> I just did a new PR with that change.

I think you meant a new commit ;-)